### ISSUE TITLE
There is no limit in recusion inside the parser.

### ISSUE THREAD

**Issue by keltia** [keltia/dices-rs#13]
You can probably easily make the parser blow up with defining an alias pointing to another one which points to the first one.  We should limit recursion to something sensible like 5.

**Comment by keltia** [comment]
Like in:
```text
dices/0.9.1 by Ollivier Robert <roberto@keltia.net>
Small CLI utility to roll dices.

Dices> list
bar = New { name: "bar", cmd: "foo" }
open = Builtin { name: "open", cmd: Open }
rulez = Alias { name: "rulez", cmd: Dice }
foo = New { name: "foo", cmd: "bar" }
list = List
mouv = New { name: "mouv", cmd: "move +7" }
roll = Alias { name: "roll", cmd: Dice }
doom = New { name: "doom", cmd: "dice 2D6" }
exit = Exit
dice = Builtin { name: "dice", cmd: Dice }
move = New { name: "move", cmd: "dice 3D6 -9" }
Dices> foo

thread 'main' has overflowed its stack
error: process didn't exit successfully: `target\debug\dices.exe -A .\testdata\aliases` (exit code: 0xc00000fd, STATUS_STACK_OVERFLOW)
```

**Comment by keltia** [comment]
Fixed in b3a54006b934ab4eab44a36c55aeeb5280e05f2f
